src/scripts/main.js

```diff
@@ -1,3 +1,320 @@
 'use strict';
 
-// write code here
+const tableHead = document.querySelector('thead');
+const tableBody = document.querySelector('tbody');
+const tableRows = tableBody.rows;
+let toggle = true;
```

Comment:
Consider using more descriptive variable names instead of 'toggle' and 'valueForSort' for better readability.